### DISCUSSION TITLE
recent showrss failures

### DISCUSSION THREAD

**Discussion by jwd3ca** [Flexget/Flexget#2905] [Q&A]
Recently, my tried and true config.yml started missing some episodes. Started with Seth Myers, and escalated to all the late night shows. Still downloads some other shows, however (This is Us, eg). 

Yesterday, when I tried a test execute, i'm now getting:

failure reason: Unexpected html content received from `https://www.tvmaze.com/episodes...
for every show. There is no mention of tvmaze in my config, or anywhere else I can find. Including the rss feed itself
( http://showrss.info/user/xxxxxx.rss?magnets=true&namespaces=true&name=null&quality=null&re=null )

Same showrss feed as always.

I can supply more info (config file. debug output, etc.) but maybe the 'tvmaze' clue is sufficient for someone to recognize the issue?

Thanks!

**Comment by ksurl** [comment]
Need to see the config.
